tests/PHPStan/Analyser/nsrt/count-chars.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
Files in nsrt/ folder should have a `// lint` comment. 

That way you also can have 2 different files for testing different php versions